src/app/app-modules/associate-anm-mo/video-consultation/video-consultation.component.ts

```
@@ -0,0 +1,273 @@
+import { MatSnackBar } from '@angular/material/snack-bar';
+import {
+  MatLegacyDialog as MatDialog,
+  MatLegacyDialogRef as MatDialogRef,
+  MAT_LEGACY_DIALOG_DATA as MAT_DIALOG_DATA,
+} from '@angular/material/legacy-dialog';
+import { AssociateAnmMoService } from '../../services/associate-anm-mo/associate-anm-mo.service';
+import { SmsTemplateService } from '../../services/smsTemplate/sms-template.service';
+import { LoginserviceService } from '../../services/loginservice/loginservice.service';
+import { map, switchMap } from 'rxjs/operators';
+
+declare var JitsiMeetExternalAPI: any;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)